runtimes/shared-configuration/src/funding.rs

```diff
@@ -22,7 +22,7 @@ use polimec_common::USD_UNIT;
 use sp_arithmetic::{FixedU128, Percent};
 use sp_std::{collections::btree_map::BTreeMap, vec, vec::Vec};
 
-#[cfg(feature = "fast-mode")]
+#[cfg(any(feature = "fast-mode", not(any(feature = "fast-mode", feature = "instant-mode"))))]
```

Comment:
I think we can simplify this as we have 3 states. Normal, fast-mode and instant-mode.
We describe normal mode as `!(fast-mode or instant-mode)`. We are currently using Hours in both normal mode and fast-mode, so only not in instant-mode. So think we can change this to `cfg(not(feature = "instant-mode"))`. 
Wdty?